### PR TITLE
[ACCSEC-19204] logging documentation

### PR DESCRIPTION
**Pull request by SergioFierro**
## Jira Ticket
- ACCSEC-19204

## Description
- Document exposed classes for logging
- Add an explanation to the README of how to use logging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +121,31 @@ In order to run the sample app, you have to create a project and application in
...
+<a name='Logging'></a>
+
+## Logging
+By default, logging is disabled. To enable it you can either set your own logging services by implementing [LogService](https://github.com/twilio/twilio-verify-android/blob/main/verify/src/main/java/com/twilio/verify/logger/LoggerService.kt) and calling `addLoggingService` (note that you can add as many logging services as you like) or enable the default logger service by calling `enableDefaultLoggingService`. Your multiple implementations and the default one can work at the same time, but you may just want to have it enabled during the development process, it's risky to have it turned on when releasing your app.
+
+### Setting Log Level
+You may want to log only certain processes that are happening in the SDK, or you just want to log it all, for that the SDK allows you to set a log level.
+* Error: reports behaviors that shouldn't be happening.
+* Info: warns specific information of what is being done.
+* Debug: detailed information.
```
Comment (sanarroya) [inline README.md:133]: I thought we were getting rid of this level

**Comment by AuthyApps** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The Pull Request summary does not contain the commit message convention</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_Linter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md (hunk 1 above)
Comment (Greshot) [reply README.md:133]: +1

Comment (yafuquen) [reply README.md:133]: We decided to remove Off, for debug the idea is to print objects (e.g. payloads) and info will be used to inform about actions happening

File: README.md [hunk 2]
```diff
@@ -120,6 +121,31 @@ In order to run the sample app, you have to create a project and application in
 * Approve or deny the challenge
 * After the challenge is updated, you will see the challenge status in the backend's `Create Push Challenge` view
 
+<a name='Logging'></a>
+
+## Logging
+By default, logging is disabled. To enable it you can either set your own logging services by implementing [LogService](https://github.com/twilio/twilio-verify-android/blob/main/verify/src/main/java/com/twilio/verify/logger/LoggerService.kt) and calling `addLoggingService` (note that you can add as many logging services as you like) or enable the default logger service by calling `enableDefaultLoggingService`. Your multiple implementations and the default one can work at the same time, but you may just want to have it enabled during the development process, it's risky to have it turned on when releasing your app.
```
Comment (yafuquen) [inline README.md:127]: Please change it to LoggerService

**Review by sanarroya — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (SergioFierro) [reply README.md]: Done

**Comment by AuthyApps** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The Pull Request summary does not contain the commit message convention</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_Linter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by yafuquen — APPROVED** [review]

**Comment by AuthyApps** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The Pull Request summary does not contain the commit message convention</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_Linter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by Greshot — APPROVED** [review]

**Pull request merged**